datafusion/src/logical_plan/expr.rs

```diff
@@ -1317,6 +1318,79 @@ pub fn normalize_cols(
         .collect()
 }
 
+/// Rewrite sort on aggregate expressions to sort on the column of aggregate output
```

Comment:
```suggestion
/// Rewrite sort on aggregate expressions to sort on the column of aggregate output
/// For example, `max(x)` is written to `col("MAX(x)")` 
```

It took me a bit of thought to understand what this was doing so I think an extra comment might help.